bazel/external/wasmedge.BUILD

```diff
@@ -0,0 +1,24 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+licenses(["notice"])  # Apache 2
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+)
+
+cmake(
+    name = "wasmedge_lib",
+    cache_entries = {
+        "WASMEDGE_BUILD_AOT_RUNTIME": "Off",
```

Comment:
Do you have plans to add AOT support as well?

Reply:
We'll keep doing this.